assets/javascripts/kitten/components/cards/contribution-card/components/action.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import classNames from 'classnames'
+import { Button } from '../../../../components/buttons/button'
+
+export const Action = props => {
+  return (
+    <Button
+      className={classNames(
+        'k-ContributionCard__action',
+        props.classNames,
+      )}
+      borderRadius={4}
+      modifier="helium"
+      {...props}
+    />
```

Comment:
Attention, pour celle-là et pour les suivantes : le `{...props}` va overrider le `props.className` s'il est défini après. Ou bien tu sépares le `className` dans les props du composant, ou bien tu le déclares après `{...props}`


```suggestion
      borderRadius={4}
      modifier="helium"
      {...props}
      className={classNames(
        'k-ContributionCard__action',
        props.classNames,
      )}
    />
```